scripts/01_IAvH_test/area_layers.yml

```diff
@@ -0,0 +1,28 @@
+script: area_layers.R
+description: "This sample script shows how it works."
+external_link: https://github.com/GEO-BON/biab-2.0
+inputs:
+  dir_stack:
+    label: dir_stack
+    description: dir_stack
+    type: text
+    example: '/output/01_IAvH_test/load_collection.R/{{dir_stack}}/output.json'
+  dir_data_layer:
+    label: dir_data_layer
+    description: dir_data_layer
+    type: text/csv
+    example: '/output/01_IAvH_test/load_collection.R/{{dir_data_layer}}/output.json'
+  group:
+    label: group
+    description: group
+    type: text
+    example: "period, classes"
+outputs:
+  dir_areas:

```

Comment:
Is it that dir_ prefix is misleading me, or this looks again like a directory? Outputs should be numbers, strings or files... If this is just a link to a CSV, please remove the dir_...